### PR TITLE
Fixed Issue #63

### PR DESCRIPTION
**Pull request by brullsker**
This fixes the scrolling issue mentioned in #63

### CODE REVIEW

File: FluentTerminal.App/Views/SettingsPages/ShellProfileSettings.xaml [hunk 1]
```diff
@@ -73,124 +73,130 @@
...
             Margin="24,0,0,0"
             Content="{x:Bind ViewModel.SelectedShellProfile, Mode=TwoWay}">
             <ContentControl.ContentTemplate>
                 <DataTemplate x:DataType="viewmodels:ShellProfileViewModel">
-                    <Grid>
-                        <StackPanel Visibility="{x:Bind InEditMode, Converter={StaticResource TrueToVisibleConverter}, Mode=OneWay}">
-                            <TextBox
+                    <Grid HorizontalAlignment="Stretch" VerticalAlignment="Stretch">
+                        <ScrollViewer HorizontalAlignment="Stretch" Visibility="{x:Bind InEditMode, Converter={StaticResource TrueToVisibleConverter}, Mode=OneWay}" VerticalScrollBarVisibility="Auto">
+                            <StackPanel Margin="0,0,20,0">
```
Comment (felixse) [inline FluentTerminal.App/Views/SettingsPages/ShellProfileSettings.xaml]: change padding to 24 to be in line with the rest of the margins/paddings

File: FluentTerminal.App/Views/SettingsPages/ShellProfileSettings.xaml [hunk 2]
```diff
@@ -73,124 +73,130 @@
...
                                 IsSpellCheckEnabled="False"
                                 Text="{x:Bind Arguments, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" />
-                        </StackPanel>
-                        <StackPanel Visibility="{x:Bind InEditMode, Mode=OneWay, Converter={StaticResource FalseToVisibleConverter}}">
-                            <TextBlock Style="{StaticResource SubtitleTextBlockStyle}" Text="{x:Bind Name, Mode=OneWay}" />
-                            <controls:HeaderedTextBlock
+                            </StackPanel>
+                        </ScrollViewer>
+                        <ScrollViewer HorizontalAlignment="Stretch" Visibility="{x:Bind InEditMode, Mode=OneWay, Converter={StaticResource FalseToVisibleConverter}}" VerticalScrollBarVisibility="Auto">
+                            <StackPanel Margin="0,0,20,0">
```
Comment (felixse) [inline FluentTerminal.App/Views/SettingsPages/ShellProfileSettings.xaml]: change padding to 24 to be in line with the rest of the margins/paddings

File: FluentTerminal.App/Views/SettingsPages/ThemeSettings.xaml [hunk 3]
```diff
@@ -169,137 +171,139 @@
             RequestedTheme="{x:Bind ViewModel.SelectedTheme.Background, Mode=OneWay, Converter={StaticResource BackgroundToApplicationThemeConverter}}">
             <ContentControl.ContentTemplate>
                 <DataTemplate x:DataType="viewmodels:ThemeViewModel">
-                    <StackPanel>
-                        <TextBlock
+                    <ScrollViewer VerticalScrollBarVisibility="Auto" Padding="0,0,20,0">
```
Comment (felixse) [inline FluentTerminal.App/Views/SettingsPages/ThemeSettings.xaml]: change padding to 24 to be in line with the rest of the margins/paddings

**Comment by felixse** [comment]
applicationframehost_2018-08-15_18-49-42

The positioning of the scrollbar in themes/profiles is a bit off, we should try to make the detail template to stretch, so that the scrollbar is on the end of the window

But other than that it looks good, thanks for your help 👍

**Comment by brullsker** [comment]
I think I've done the changes you requested.
Besides, regarding the position of the ScrollBar in Themes / Profiles I totally agree with you.

I hope I can help again on some other issues.

**Comment by felixse** [comment]
Was no big issue, I just pushed it into this PR.
Looks goods now, thank you 😃

**Pull request merged**